DHIS2/post_clone_scripts/file_garbage_remover.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+import argparse
+
+
+LOCAL_SQL_FILE = "/tmp/find_orphan_files.sql"
+LOCAL_LIST_FILE = "/tmp/list_of_files_to_be_removed.txt"
+REMOTE_LIST_FILE = "/tmp/list_of_files_to_be_removed.txt"
+FILE_BASE_PATH = "/DHIS2_home/files"
+
+
+SQL_QUERY = """
+    SELECT storagekey
+    FROM fileresource fr
+    WHERE NOT EXISTS (
+        SELECT 1 FROM document d WHERE d.fileresource = fr.fileresourceid
+    )
+    AND fr.uid NOT IN (
+        SELECT url FROM document
+    )
+    AND fr.domain = 'DOCUMENT';
+"""
+
+
+def run(command, check=True, capture=False):
+    """Run a shell command."""
+    print(f"$ {' '.join(command)}")
+    return subprocess.run(command, check=check, capture_output=capture, text=True)
+
+
+def find_container_id(name_pattern):
+    """Find the container ID based on a name substring."""
+    result = run(["docker", "ps", "--format", "{{.ID}}\t{{.Names}}"], capture=True)
+    for line in result.stdout.strip().splitlines():
+        cid, name = line.split("\t")
+        if name_pattern in name:
+            return cid
+    return None
```

Comment:
To avoid having to manually filter, just ask the command to give us the filtered list
```suggestion
    """Find the container ID based on a name substring. Retrieves first match only"""
    result = run(["docker", "ps", "--format", "{{.ID}}", "-f", f"name={name_pattern}"], capture=True)
    lines = result.stdout.strip().splitlines()
    return lines[0] if lines else None
```